engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -345,6 +441,15 @@ public static String removeComments(String s) {
         return s.trim();
     }
 
+    /**
+     * Represents an import that can be added to an ImportCustomizer, as a valid return from
+     * {@link #isValidImportString(String)}.
+     */
+    @VisibleForTesting
+    public interface GroovyImport {
+        void append(ImportCustomizer imports);
```

Comment:
I was confused at first trying to figure out how `isValidImportString` works now. I think the method name `appendTo` makes more sense?

Reply:
Agreed, just posted my own naming thoughts for the isValidImportString method itself, will change this as well.